.github/workflows/deploy_do.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy to DigitalOcean
+
+on:
+  push:
+    branches: [$default-branch]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out Github repository
+        uses: actions/checkout@v2
+      - name: Install doctl
+      uses: digitalocean/action-doctl@v2
+      with:
+        token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
```

Comment:
Where does this workflow get this secret from? Anyone pushing to our main can and will deploy automatically to digital ocean?

Reply:
Btw, Any merge to master or to any branch?

Reply:
The secret is from [actions secrets](https://github.com/42AGV/ft_transcendence/settings/secrets/actions).
I think the action runs when we merge to the main branch since we can't push directly with the current settings.

Reply:
There seem to be a better way to trigger the action -> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges
I will change it :+1:

Reply:
Updated, now it runs when a PR to the main branch is closed and merged. Thanks for the suggestions.